src/content/docs/blog/roadmap-2025.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Roadmap 2025 and Biome 2.0
+summary: "A look at what 2025 will bring for us: Biome 2.0, enterprise support, and our roadmap"
+authors:
+  - arendjr
+  - team
+date: 2025-01-20
+cover:
+  light: "@/assets/blog/roadmap-2024/banner-light.png"
+  dark: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: "Biome - Toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+Today we're happy to share our plans for Biome 2.0 as well as the rest of our roadmap for 2025. But before we dive into what's coming, let's do a quick recap of the major developments in 2024.
+
+## 🎆 Recap: Biome in 2024
+
+2024 was a great year for Biome. Let's see what happened:
+
+* We released 4 new "minor" Biome versions, from 1.6 through 1.9, with plenty of useful features:
+  * New `biome search` and `biome explain` commands, while the `biome migrate` command was significantly expanded to help users coming from ESLint and Prettier.
+  * Added support for **CSS** and **GraphQL** formatting and linting. 
+  * Partial support for **Astro**, **Svelte** and **Vue** files.
+  * The ability to let configuration files extend from one another, which is especially useful in monorepo and larger organizational setups.
+  * We added countless new lint rules and miscellaneous fixes and improvements.
```

Comment:
I would add the following:
- `.editorconfig` support
- a specific mention for [`useSortedClasses`](https://biomejs.dev/linter/rules/use-sorted-classes/) (maybe with the link to the issue). Our users requested tailwind support, so maybe we could mention that we **started** the support for tailwind, and we would like to complete it with the user's help
- Reporters! :) https://biomejs.dev/reference/reporters/